src/ape_run/_cli.py

```diff
@@ -252,6 +252,7 @@ def _launch_console(self):
     default=False,
     help="Drop into interactive console session after running",
 )
+@network_option()
```

Comment:
this isn't breaking right?

Reply:
oops, this was research on something else, didnt mean to commit